src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -349,5 +403,15 @@ export default compose(
                 return `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`;
             },
         },
+        transactionViolation: {
+            key: ({report}) => {
+                const parentReportAction = ReportActionsUtils.getParentReportAction(report);
```

Comment:
@trevor-coleman `ReportActionsUtils.getParentReportAction` is a deprecated method. I'm trying to remove these in https://github.com/Expensify/App/issues/27262. Can you please create a PR to clean this up and remove the usage of the deprecated method?

Reply:
Hi, I was a paid contractor and I'm no longer on this project.

Reply:
@tgolen can you do this as part of one of your PRs removing the method?

Reply:
I'll add it to the list.

Thanks for responding @trevor-coleman and I hope you come back at some point! 👋

Reply:
Thanks!